public/iiif/speculative-annotations-manifest.json

```diff
@@ -4868,7 +4879,7 @@
       ],
       "homepage": [
         {
-          "id": "__URL__",
+          "id": "https://www.loc.gov/item/ihas.200218413/",
           "type": "Text",
           "label": { "en": ["Home page for __WORK_NAME__"] },
```

Comment:
@jaimemears Do you want to update `__WORK_NAME__` here as well?

Reply:
Yes, that makes sense. The protocol is the title corect? so it would be ["Home page for [Candide: Candide green and purple cape/vest]"]

Reply:
Yes, correct that looks good to me, though you can title it however you like.

Reply:
updated in latest commit